### PR TITLE
[MODULAR] Table flipping now throws anything on it in the direction it was flipped

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Title. Had this idea while fixing table flipping.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

https://github.com/Skyrat-SS13/Skyrat-tg/assets/59709059/4fd7a88f-8679-4953-84d8-c57db7d7997a

</details>

## Changelog

:cl:
add: Table flipping now throws everything on the table
/:cl:

### CODE REVIEW

**Comment by W4rd3nn** [comment]
speedmerge NOW

File: modular_skyrat/modules/tableflip/code/flipped_table.dm [hunk 1]
```diff
@@ -94,6 +94,17 @@
...
+	var/turf/throw_target = get_step(flipped_table, flipped_table.dir)
+	if (!isnull(throw_target))
+		for (var/atom/movable/movable_entity in flipped_table.loc)
+			if (movable_entity == flipped_table)
+				continue
+			if (movable_entity.anchored)
+				continue
+			if (movable_entity.invisibility > SEE_INVISIBLE_LIVING)
+				continue
+			movable_entity.safe_throw_at(throw_target, range = 1, speed = 1, force = MOVE_FORCE_NORMAL, gentle = TRUE)
```
Comment (vinylspiders) [inline modular_skyrat/modules/tableflip/code/flipped_table.dm:120]:
```suggestion
			if(!ismob(movable_entity) && !isobj(movable_entity))
				continue
			if(movable_entity.throwing || (movable_entity.movement_type & (FLOATING|FLYING)))
				continue
			movable_entity.safe_throw_at(throw_target, range = 1, speed = 1, force = MOVE_FORCE_NORMAL, gentle = TRUE)
```
Love this but I worry that we might accidentally move some things that are not meant to see the light of day. Best add some additional protections. Also there is floaty stuff.

**Review by vinylspiders — CHANGES_REQUESTED** [review]

**Comment by vinylspiders** [comment]
This is optional obviously but it might be funny to add a little description of the pacifist gently turning the table over (and either not playing the sound or making it low volume) lol

**Review by vinylspiders — APPROVED** [review]

**Comment by nikothedude** [comment]
sure

**Pull request merged**